packages/containers_common.rb

```diff
@@ -0,0 +1,116 @@
+# Adapted from Arch Linux containers-common PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/containers-common/trunk/PKGBUILD
+
+require 'package'
+
+class Containers_common < Package
+  description 'Configuration files and manpages for containers'
+  homepage 'https://github.com/containers'
+  version '0.47.4'
+  license 'Apache'
+  compatibility 'all'
+  source_url 'https://github.com/containers/common.git'
+  git_hashtag "v#{version}"
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containers_common/0.47.4_armv7l/containers_common-0.47.4-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containers_common/0.47.4_armv7l/containers_common-0.47.4-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containers_common/0.47.4_x86_64/containers_common-0.47.4-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '653d744a20a0d5be563586212b7813887b15b4e56b8c6344a078fa99e764e6f9',
+     armv7l: '653d744a20a0d5be563586212b7813887b15b4e56b8c6344a078fa99e764e6f9',
+     x86_64: '21dfc889cc2fcd41ff44ce30c7aa900d2ffa0a3d9a48f22b732ddbeae0c52afe'
+  })
+
+  depends_on 'netavark'
+  depends_on 'go_md2man' => ':build'
+
+  def self.build
+    @image_version = 'v5.20.0'
+    @podman_version = 'v4.0.2'
+    @shortnames_version = 'v2022.02.08'
+    @skopeo_version = 'v1.6.1'
+    @storage_version = 'v1.38.2'
+    Dir.chdir 'docs' do
+      system 'for _man_page in *.md
+      do
+        go-md2man -in $_man_page -out ${_man_page//.md}
```

Comment:
Did you mean `${_man_page/.md/}`?

Reply:
Copy from https://github.com/archlinux/svntogit-community/raw/packages/containers-common/trunk/PKGBUILD

Reply:
Thoughts on that?